Test/AccountTest.cs

```diff
@@ -146,5 +147,19 @@ public void GetAccountNotes()
 
             notes.Should().BeEmpty();
         }
+
+        [RecurlyFact(TestEnvironment.Type.Integration)]
+        public void Balance()
```

Comment:
Well, I think it won't send that field unless it applies. I'm not sure I can think of a way to make an account in a past due state. I'll look into it tomorrow.